templates/skeleton/app/routes/account._index.tsx

```diff
@@ -1,7 +1,8 @@
-import type {LoaderFunctionArgs} from '@shopify/remix-oxygen';
-
-export async function loader({response}: LoaderFunctionArgs) {
-  response!.status = 302;
-  response!.headers.set('Location', '/account/orders');
-  return null;
```

Comment:
using loader (maybe also because of resource route?) while manipulating headers cause error